### PR TITLE
[DOC] Updated docs for multiclass

### PR DESCRIPTION
**Pull request by outoftardis**
# Description
- Added resultsToEvaluate parameter and output section to multiclass classifier
- Aligned markup for logistic regression and multiclass classifier

image

### CODE REVIEW

File: docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst [hunk 1]
```diff
@@ -199,6 +199,43 @@ At the prediction stage, a multi-class classifier has the following parameters:
...
+
+.. list-table::
+   :widths: 10 60
+   :header-rows: 1
+   :align: left
+
+   * - Result ID
+     - Result
+   * - ``decisionFunction``
+     - A numeric table of size :math:`n \times \frac{K(K-1)}{2}` containing the results of the decision function
```
Comment (PetrovKP) [inline docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst:228]: [Here ](http://oneapi-src.github.io/oneDAL/dev_guide/algorithms/svm_multi_class/multi-class-classifier.html#training-stage) - stage use slightly different style 
image

I would suggest aligning to either the first case or the second

**Review by PetrovKP — APPROVED** [review]

File: docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst (hunk 1 above)
Comment (outoftardis) [reply docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst:228]: done

File: docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst [hunk 2]
```diff
@@ -158,7 +158,7 @@ Prediction
 At the prediction stage, a multi-class classifier has the following parameters:
 
 .. list-table::
-   :widths: 10 20 30 25
+   :widths: 10 10 10 30
```
Comment (averbukh) [inline docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst:161]: Could you please fix following as well in this table? 
" -  ``voteBasedthe`` method based on the votes obtained from two-class classifiers."

Comment (outoftardis) [reply docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst:161]: thanks, fixed

**Comment by outoftardis** [comment]
@napetrov please confirm that it's ok to merge (that the ci is not failing because of my changes)

**Pull request merged**